src/views/menus/AccountMenu.tsx

```diff
@@ -219,6 +227,18 @@ export const AccountMenu = () => {
           label: stringGetter({ key: STRING_KEYS.DISPLAY_SETTINGS }),
           onSelect: () => dispatch(openDialog({ type: DialogTypes.DisplaySettings })),
         },
+        ...(isDev
+          ? [
+              {
+                value: 'ComplianceConfig',
+                icon: <Icon iconName={IconName.Gear} />,
+                label: 'Compliance Config',
```

Comment:
can we link a todo to localize this?

Reply:
I figure it is ok to not localize since it is for dev only.